software/firmware/experimental/custom_font.md

```diff
@@ -0,0 +1,61 @@
+Custom Font support
+===================
+
+The `CustomFontWriter` class is adapted from https://github.com/peterhinch/micropython-font-to-py/tree/master/writer.  
+
+A child class to the one found in `europi.py`, named `Display`, is created and named `CustomFontDisplay`.  
+
+To use custom/large fonts, oled must be redefined as a `CustomFontDisplay` rather than a plain `Display`, but this is done for you as soon as you import `custom_font.py`.
+
+How to generate custom fonts :
+------------------------------
+
+To generate font files, use https://github.com/peterhinch/micropython-font-to-py/blob/master/font_to_py.py.
+
+The fonts in this folder have been generated from TTF files downloaded from with https://www.wfonts.com/.
+
+The commands are :
+
+    python3 font_to_py.py FreeSans.ttf 14 freesans14.py -x
+    python3 font_to_py.py FreeSans.ttf 17 freesans17.py -x
+    python3 font_to_py.py FreeSans.ttf 20 freesans20.py -x
+    python3 font_to_py.py FreeSans.ttf 24 freesans24.py -x
+
+The -x option is important to have the font horizontally mapped.
+
+How to use the CustomFontWriter class :
+---------------------------------------
+
+See `software/contrib/custom_font_demo.py` for usage examples.
+
+To test the CustomFontWriter in your code you can simply replace :
+
+    from europi import oled
+
+with :
+
+    from experimental.custom_font import oled
+
+By default, custom_font.Display will use the 8x8 pixel monospaced font.
```

Comment:
[optional] It's good practice to use a monospaced font when referencing code or class names in documentation. It helps the reader separate the code from the prose. One way to do this is with double backticks, as in:

```By default, ``custom_font.Display`` will use the```

There are several places in this document where this can be used.